pkg/types/order.go

```diff
@@ -260,6 +260,7 @@ type OrderQuery struct {
 	Symbol        string
 	OrderID       string
 	ClientOrderID string
+	UUID          string
```

Comment:
call it OrderUUID? Align with the OrderID field

Reply:
done